core/src/epicli/data/common/ansible/playbooks/roles/filebeat/templates/filebeat.yml.j2

```
@@ -180,14 +180,14 @@ setup.template.settings:
 # the dashboards is disabled by default and can be enabled either by setting the
 # options here or by using the `setup` command.
 {% set dashboards_enabled = is_upgrade_run | ternary(specification.kibana.dashboards.enabled | default(False), specification.kibana.dashboards.enabled) %}
```

```suggestion
{% set dashboards_enabled = is_upgrade_run | ternary(existing_setup_dashboards.enabled, specification.kibana.dashboards.enabled) %}
```
It seems that for upgrade mode we always have this setting set to `auto` because `manifest.yml` in `var` directory contains settings merged with current defaults. We may need refactoring to disable such merging and separate current defaults into dedicated file or allow overriding the default value via the config file. IMO loading vars explicitly is less error prone since gives you control over the source of data. Until the refactoring is done I think we should avoid complex flow, thus my suggestion is to preserve the old setting (`false`) for now. If someone needs this functionality they may run `apply` after `upgrade`.

Note: `existing_setup_dashboards` has to be added in `core\src\epicli\data\common\ansible\playbooks\roles\upgrade\tasks\filebeat.yml`.